Mathlib/Analysis/LocallyConvex/Basic.lean

```diff
@@ -103,6 +103,11 @@ theorem balanced_iInter₂ {f : ∀ i, κ i → Set E} (h : ∀ i j, Balanced 
     Balanced 𝕜 (⋂ (i) (j), f i j) :=
   balanced_iInter fun _ => balanced_iInter <| h _
 
+theorem Balanced.mulActionHom_preimage [SMul 𝕜 F] {s : Set F} (hs : Balanced 𝕜 s)
+    (f : E →ₑ[(RingHom.id 𝕜)] F) : Balanced 𝕜 (f ⁻¹' s) := fun a ha x ⟨y,⟨hy₁,hy₂⟩⟩ => by
```

Comment:
```suggestion
    (f : E →[𝕜] F) : Balanced 𝕜 (f ⁻¹' s) := fun a ha x ⟨y,⟨hy₁,hy₂⟩⟩ => by
```

Reply:
@j-loreaux That's what I originally tried, by then in the proof of `LinearMap.polar_AbsConvex`, `Balanced.mulActionHom_preimage` doesn't accept the `B i` argument with the message:
```
application type mismatch
  Balanced.mulActionHom_preimage balanced_closedBall_zero (B i)
argument
  B i
has type
  F →ₗ[𝕜] 𝕜 : Type (max u_3 u_1)
but is expected to have type
  F →ₑ[_root_.id] 𝕜 : Type (max u_3 u_1)
```

Reply:
Okay, I think I have this figured out now.